plugins/modules/rds_cluster.py

```diff
@@ -171,11 +171,29 @@
     engine:
         description:
           - The name of the database engine to be used for this DB cluster. This is required to create a cluster.
+          - The combinaison of I(engine) and I(engine_mode) may not be supported.
+          - "See AWS documentation for details:
+            L(Amazon RDS Documentation,https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_CreateDBCluster.html)."
         choices:
           - aurora
           - aurora-mysql
           - aurora-postgresql
+          - mysql
+          - postgres
```

Comment:
I guess these two can be removed from this PR since they require other parameters we do not support and a specific version of botocore.

Reply:
Agreed, and you've got them covered with your own PR.